tests/queries/0_stateless/01600_detach_permanently.sh

```diff
@@ -13,7 +14,7 @@ mkdir -p "${WORKING_FOLDER_01600}"
 clickhouse_local() {
     local query="$1"
     shift
-    ${CLICKHOUSE_LOCAL} --query "$query" $@ -- --path="${WORKING_FOLDER_01600}"
+    ${CLICKHOUSE_LOCAL} --query "$query" "$@" -- --path="${WORKING_FOLDER_01600}"
```

Comment:
it's not the same. I guess it will fail.

Reply:
I guess this test is green, so it works?

Reply:
Ok, let's keep it. Since the test is happy I'm also happy :)